Majasdarbs/02-majasdarbs/Form/form.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Reģistrācijas forma</title>
+  </head>
+  <body>
+    <h2>Reģistrācijas forma</h2>
+    <!-- Forma -->
+    <form action="" method="get">
+      <fieldset>
+        <label for="name">Ievadīt vārdu:</label>
+        <input type="text" name="name" />
+        <label for="password">Ievadīt paroli:</label>
+        <input type="password" name="password" />
+      </fieldset>
+      <br />
+      <fieldset>
+        <legend>Informācija par sevi:</legend>
+        <h4>Dzimums</h4>
+        <div>
+          <input type="radio" name="man-sex" />
+          <label for="man-sex">Vīrietis</label>
+        </div>
+        <div>
+          <input type="radio" name="woman-sex" />
+          <label for="woman-sex">Sieviete</label>
```

Comment:
Name should be equal for all. Now you can choose both (man and woman), but should only one.